packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js

```diff
@@ -1,6 +1,8 @@
 import {
   checkMaxLength, isEncoded, checkValidNrrFormat, checkRoundToNearest, checkIgnoredParameter,
   isReserved, isPositive, checkBlKey, checkCorrectType, checkOtValidValue, checkSfValidValue,
+  checkPRValue,
```

Comment:
I would change checkPRValue to checkPrValue to be consistent with the other names